SPECS/azurelinux-sysinfo/azurelinux-sysinfo.spec

```diff
@@ -38,6 +38,35 @@ install -m 755 %{SOURCE2} %{buildroot}%{_sysconfdir}/systemd/system/
 %{_sysconfdir}/systemd/system/azurelinux-sysinfo.service
 
 %post
+# Apply required SElinux policies only if selinux-policy is present
+if rpm -q selinux-policy &> /dev/null; then
+    mkdir -p /tmp/
+    POLICYFILE=/tmp/sysinfo-selinuxpolicies.cil
+    cat << EOF > $POLICYFILE
+(allow systemd_analyze_t sysctl_kernel_t (dir (search)))
+(allow systemd_analyze_t locale_t (dir (search)))
+(allow systemd_analyze_t init_runtime_t (dir (search)))
+(allow systemd_analyze_t sysctl_kernel_t (file (read)))
+(allow systemd_analyze_t locale_t (file (read)))
+(allow systemd_analyze_t systemd_analyze_t (capability (net_admin)))
+(allow systemd_analyze_t init_t (unix_stream_socket (connectto)))
+(allow systemd_analyze_t system_dbusd_runtime_t (dir (search)))
+(allow systemd_analyze_t security_t (filesystem (getattr)))
+(allow systemd_analyze_t selinux_config_t (dir (search)))
+(allow systemd_analyze_t init_t (system (status)))
+(allow systemd_analyze_t init_t (service (status)))
+(allow systemd_analyze_t systemdunit (service (status)))
+(allow systemd_analyze_t etc_t (service (status)))
+
+EOF
+
+    # Change SELinux context of file to user_tmp_t so that it can be read by semanage_t
+    chcon -t user_tmp_t $POLICYFILE
+    # Apply the SELinux policies
+    semodule -i $POLICYFILE
+fi
+
+# Script to enable the systemd service
 #!/bin/sh
```

Comment:
Move this to just after `%post` line?

Reply:
why? it's still in the post section

Reply:
If you want the entire post script to be run with /bin/sh, it would be better to move the choice of the shell to the top of the script.
https://www.linuxfoundation.org/blog/blog/classic-sysadmin-writing-a-simple-bash-script#:~:text=The%20%E2%80%9C%23!%E2%80%9D%20combo%20is%20called,that%20actually%20runs%20the%20script.

Reply:
Ohhh, I thought you meant moving the enable script up

Reply:
moved